packages/babel-cli/src/babel/index.js

```diff
@@ -74,6 +74,7 @@ commander.option("-o, --out-file [out]", "Compile all input files into a single
 commander.option("-d, --out-dir [out]", "Compile an input directory of modules into an output directory");
 commander.option("-D, --copy-files", "When compiling a directory copy over non-compilable files");
 commander.option("-q, --quiet", "Don't log anything");
+commander.option("-l, --delete-dir-on-start", "Delete's the out directory before compilation");
```

Comment:
@keshcode re: CI failures, have you looked at https://github.com/babel/babel/blob/7.0/packages/babel-core/src/config/option-manager.js#L29?